src/API/FeedState.php

```diff
@@ -465,21 +446,25 @@ private static function map_status_into_extra_info( array $processing_results ):
 		switch ( $status ) {
 			case 'COMPLETED':
 			case 'COMPLETED_EARLY':
-			case 'PROCESSING':
 				return sprintf(
 					/* Translators: %1$s Time difference string, %2$s number of products */
-					esc_html__( 'Last pulled: %1$s ago, containing %2$d products', 'pinterest-for-woocommerce' ),
+					esc_html__( '%1$s ago, containing %2$d products', 'pinterest-for-woocommerce' ),
 					human_time_diff( $processing_date->getTimestamp() ),
 					$original_products_count
 				);
 			case 'FAILED':
 				$info = sprintf(
 					/* Translators: %1$s Time difference string */
-					esc_html__( 'Last attempt: %1$s ago', 'pinterest-for-woocommerce' ),
+					esc_html__( '%1$s ago', 'pinterest-for-woocommerce' ),
 					human_time_diff( $processing_date->getTimestamp() )
 				);
 				$global_error = Pinterest\FeedStatusService::get_processing_results_global_error( $processing_results );
 				return $info . ( $global_error ? ' - ' . $global_error : '' );
+			case 'PROCESSING':
+				return sprintf(
+					/* Translators: %1$s Time difference string, %2$s number of products */
+					esc_html__( 'overview numbers can be inaccurate untill processing finishes.', 'pinterest-for-woocommerce' ),
```

Comment:
Edit for the notice sentence:

```suggestion
					esc_html__( 'Overview numbers may be inaccurate until processing finishes.', 'pinterest-for-woocommerce' ),
```